src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -862,23 +849,23 @@ sealed class RsDiagnostic(
             errorCode: RsErrorCode,
         ) : this(element, itemNamespace.itemName, itemName, scope.formatScope(), errorCode)
 
-        override fun prepare() = PreparedAnnotation(ERROR, errorCode, errorText())
-
-        private fun errorText(): String = when {
-            element.ancestorOrSelf<RsUseSpeck>() != null ->
-                "A second item with name `$itemName` imported. Try to use an alias."
-            errorCode == E0259 ->
-                "A second extern crate with name `$itemName` imported"
-            else ->
-                "A $itemType named `$itemName` has already been defined in this $scopeType"
-        }
+        // TODO: provide quick fixes:
+        //  - add type alias for `use` and `extern crate` items
+        //  - navigate to/show other items
+        override fun prepare(): PreparedAnnotation = PreparedAnnotation(
+            ERROR,
+            errorCode,
+            "The name `$itemName` is defined multiple times",
+            "`$itemName` must be defined only once in the $itemType namespace of this $scopeType"
+        )
 
         companion object {
             private fun PsiElement.formatScope(): String =
                 when (this) {
                     is RsBlock -> "block"
                     is RsMod, is RsForeignModItem -> "module"
                     is RsTraitItem -> "trait"
+                    is RsEnumItem -> "enum"
```

Comment:
This should be changed to handle `RsEnumBody`, and please add corresponding test

Reply:
Nice catch!
Unfortunately, I don't know how to properly check the whole tooltip text. There is `tooltip` parameter that I can put into `error` tag, but since we add some html tags into tooltip text, test framework fails to find error markup